The Basic Building Blocks of Elm.md

```diff
@@ -101,7 +99,7 @@ Good luck!
 
 ### <input type="checkbox"> Step 5
 
-Just like Ruby has hashes and JavaScript has objects, Elm has **records** to store key-value pairs. Unlike those other languages, though, Elm does not allow you to change the number of keys of a record after creation. A record with three keys always has three keys, no matter what.
+Just like Ruby has hashes and JavaScript has objects, Elm has **records** to store key-value pairs. Unlike those other languages, though, Elm treats the keys of records as immutable. Once a record has been defined, you cannot add, edit, or remove its keys.
```

Comment:
💭  still wonder if the framing "records are like hashes, but Elm places restrictions" can be misleading

- "So Elm doesn't have 'mutable' dicts" (yes, in fact)
  - "Just like JavaScript has objects"?
- To me, the semantics of a "key" is different from the name of a field of an Elm record: key is an independent object you supply to a hash-like object to get its corresponding value
  - Say "name-value pairs" or "fields"?